.github/workflows/ci-extends.yml

```diff
@@ -0,0 +1,81 @@
+---
+
+name: ci-extends
+
+on:
+  workflow_dispatch:
+    inputs:
+      saucelab_test:
+        description: 'Enable run a Sauce lab test'
+        default: true
+        required: false
+        type: boolean
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**.md'
+      - '**.asciidoc'
+  pull_request:
+    branches:
+      - main
```

Comment:
As mentioned earlier, this won't work on PRs coming from forked repositories.

Reply:
Let's remove the `pull_request` trigger and rely on checks as gatekeepers if needed.
We can trigger the workflow manually using the `workflow_dispatch` event.